market/connector/currencyConverter/demo/currencyConverter/src_hd/currencyConverter/currencyDropdown/currencyDropdown.xhtml

```diff
@@ -0,0 +1,52 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"      
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:cc="http://xmlns.jcp.org/jsf/composite"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<cc:interface componentType="IvyComponent">
+</cc:interface>
+
+<cc:implementation>
+	<p:selectOneMenu required="true" value="#{data.selectedCurrency}" placeholder="Currency">
```

Comment:
good first solution: for a next version we should prefer to move these speaking labels into the 'openapi.json' descriptor. So that the descriptors are also visible when using the RestClientCall activity directly. Furthermore we gain flexibility to adapt new currencies by maintaining them in a single location.